playwright/all_tests/cas_auth/login.spec.ts

```diff
@@ -10,9 +10,7 @@ test('yllapitaja can login and logout', async ({ page }) => {
   await page.locator('#username').fill(process.env.TESTIKAYTTAJA_YLLAPITAJA_USERNAME!)
   await page.locator('#password').fill(process.env.TESTIKAYTTAJA_YLLAPITAJA_PASSWORD!)
   await page.locator('input[type=submit]').click()
-  await expect(page.getByTestId('page-heading-etusivu')).toContainText(
-    'Hei Ludos Ylläpitäjä, tervetuloa Koepankin ylläpitoon!'
-  )
+  await expect(page.getByTestId('page-heading-etusivu')).toContainText('Hei Ludos, tervetuloa LUDOS -palveluun!')
```

Comment:
Väli pois, ja miinus-merkki korvattu yhdysmerkillä.

```suggestion
  await expect(page.getByTestId('page-heading-etusivu')).toContainText('Hei Ludos, tervetuloa LUDOS–palveluun!')
```